packages/actor-query-operation-minus/lib/ActorQueryOperationMinus.ts

```diff
@@ -0,0 +1,64 @@
+import {AbstractBindingHash, IActorInitRdfDereferencePagedArgs} from "@comunica/actor-abstract-bindings-hash";
+import {
+    ActorQueryOperation, Bindings, BindingsStream,
+    IActorQueryOperationOutputBindings,
+} from "@comunica/bus-query-operation";
+import {ActionContext} from "@comunica/core";
+import * as RDF from "rdf-js";
+import {Algebra} from "sparqlalgebrajs";
+
+/**
+ * A comunica Minus Query Operation Actor.
+ */
+export class ActorQueryOperationMinus extends AbstractBindingHash<Algebra.Minus> {
+
+  private hashes: {[id: string]: boolean} = {};
+  private commons: string[];
```

Comment:
These two fields should not be part of this class, as we want actors to be stateless. 
Instead, these fields should be moved inside the closure of `newHashFilter` or `runOperation `.